message PBPuncturedEdge {
  required uint32 id = 1;
  required sint32 chirality = 2;
  optional float t = 3;
};

message PBPuncturedEdges {
  repeated PBPuncturedEdge edges = 2;
};

////
message PBPuncturedFace {
  required uint32 id = 1;
  required sint32 chirality = 2;
  optional float x = 3;
  optional float y = 4;
  optional float z = 5;
};

message PBPuncturedFaces {
  repeated PBPuncturedFace faces = 1;
  optional int32 timestep = 2;
};

